warewulf.spec.in

```diff
@@ -196,6 +196,8 @@ getent group %{wwgroup} >/dev/null || groupadd -r %{wwgroup}
 %dir %{_sharedstatedir}/warewulf/overlays/host
 %dir %{_sharedstatedir}/warewulf/overlays/wwinit
 %attr(-, root, root) %{_sharedstatedir}/warewulf/overlays/*/rootfs
+%attr(700, root, root) %{_sharedstatedir}/warewulf/overlays/generic/rootfs/root
```

Comment:
as far as I can tell, different distributions are using different permissions somewhere between `550` and `700`, I think all should be fine and equivalent